data/human/news.txt

```diff
@@ -1198,6 +1270,42 @@ news "human farming in pirate space"
 
 
 
+news "syndicate pa system"
+	location
+		government "Syndicate"
+		not attributes station
+	name
+		word
+			"Public Announcement System"
+	message
+		word
+			"Attention, baggage should not be left unattended. Any unattended baggage will be auctioned within days."
+			"Attention, factory workers please proceed to the fast-lane immigation desk. Time spent in regular immigration lanes will not be re-imbursed."
+			"Dear spaceship captains, please be aware that our Outfit shops are open all hours of the day. There is never a reason to delay the purchase of your next syndicate equipment."
+			"Dear traveller, we recommend you to have dinner in restaurant The Wing, you will not be disapponted."
+
+
+
+news "touristic pa system"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "deep" "rich" "paradise" "tourism"
+	name
+		word
+			"Public Announcement System"
+	message
+		word
+			"Attention, baggage should not be left unattended. Keep an eye on your belongings at all times."
+			"Attention, note to travellers; the planetary security level has been changed to black."
+			"Be aware that parking restrictions are in effect around the spaceport. Don't leave your personal transport vehicles unattended."
+			"Dear travellers, the spaceport is a strictly no-smoking zone. Thank you for your consideration."
+			"Dear travellers, welcome to our planet. Please remember to synchronize your personal clocks to the local time and to the local night/day cycle duration."
+			"Please stand clear of the doors when the shuttle-train is departing."
+			"This is the final boarding call for passengers Eline, John and Robert. Please proceed to platform 3 for immediate boarding."
```

Comment:
No last names?

Reply:
Good point. I didn't think of a good last name yet. Do you have any suggestions?

Reply:
https://lastnamegenerator.com/common-last-names

Reply:
Thanks! I added a last name.